tests/python/frontend/onnx/test_forward.py

```diff
@@ -1927,8 +1927,12 @@ def verify_binary_ops(op, x, y, out_type="float32"):
     verify_binary_ops("Sum", x, z)
     verify_binary_ops("Greater", x, y, "bool")
     verify_binary_ops("Greater", x, z, "bool")
+    verify_binary_ops("GreaterOrEqual", x, y, "bool")
```

Comment:
Can you uncomment the appropriate tests in `unsupported_onnx_tests` in this file?